components/org.wso2.carbon.appmgt.gateway/src/main/java/org/wso2/carbon/appmgt/gateway/handlers/security/saml2/SAML2AuthenticationHandler.java

```diff
@@ -649,7 +675,7 @@ private VerbInfoDTO getVerbInfoForApp(String webAppContext, String webAppVersion
      * @return true if JWT is enabled in App Manager config.
      */
     private boolean isJWTEnabled(){
```

Comment:
Need to refactor. make isJWTEnabled boolean instance variable and initialize with init method. use it for further decision making.